papas/mergedclusterbuilder.py

```diff
@@ -63,28 +78,30 @@ def __init__(self, clusters, ruler, history_nodes = None):
     def _make_merged_clusters(self):
         #carry out the merging of linked clusters
         for subgraphids in self.subgraphs:
-            if len(subgraphids)==1 : 
-                #no overlapping cluster so no merging needed
-                self.merged[subgraphids[0]] = self.clusters[subgraphids[0]]
-            else: 
-                #make a merged "supercluster" and then add each of the linked clusters into it                
-                supercluster = None
+            subgraphids.sort()
+            if len(subgraphids)==5:
+                pass
+            #make a mergecluster based on the first cluster
+            id =  subgraphids[0] # first id in list
```

Comment:
first = None 
for id in subgraphids: 
   if not first: 
       first = id 
   else